packages/ember-metal/lib/property_get.js

```diff
@@ -89,6 +89,22 @@ export function _getPath(root, path) {
   return obj;
 }
 
+function isGettable(obj) {
+  if (obj == null) {
+    return false;
+  }
+
+  let allowableTypes = ['object', 'function', 'string'];
```

Comment:
Just pushed the update. In the update I am casting `undefined` into `false`, so let me know if you believe that to be superfluous and I will remove.